pilot/cmd/pilot-agent/main.go

```diff
@@ -58,6 +58,7 @@ var (
 	controlPlaneAuthPolicy string
 	customConfigFile       string
 	proxyLogLevel          string
+	concurrency            int
```

Comment:
An alternative would be to add a 'extraFlags' string covering any other 'advanced' settings/tuning. 

However I think it's good to add an explicit concurrency flag - maybe in future the injector could fill it in based on the CPU allocation or some annotations.